.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -34,37 +32,37 @@ body:
     attributes:
       label: Bokeh version
       description: >-
-        Version of Bokeh, as provided by `bokeh info`
+        Version of Bokeh, as provided by `bokeh info`.
```

Comment:
What if we just have one text field and require them to paste all of `bokeh info` instead of separate fields?

Reply:
Yes, I was going to suggest that - I'll update the form!

Reply:
I put that field in the form. Which reminded me: `bokeh info` currently does not output information on the operating system it runs on. Would it make sense to add that to the command?

Reply:
certainly seems reasonable to me

Reply:
I made PR https://github.com/bokeh/bokeh/pull/11797 to add OS info to the command